devto/articles_test.go

```diff
@@ -3,14 +3,17 @@ package devto
 import (
 	"context"
 	"encoding/json"
+	"fmt"
 	"io/ioutil"
 	"net/http"
 	"reflect"
+	"strconv"
 	"strings"
 	"testing"
 	"time"
 
 	"github.com/VictorAvelar/devto-api-go/testdata"
+	"github.com/gorilla/mux"
```

Comment:
Do you think is appropriate to pull a new dependency just for mocking a test method?

Reply:
What I was thinking was that this would be a dependency that only lives in the client's test binary. However, I don't have a strong opinion on this, and it wouldn't be too hard to just make a helper function to get the n-th path param from an HTTP path and parse it as an int

Reply:
I think we should avoid bringing in new dependencies unless is necessary for the project.